specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-06-01-preview/mfe.json

```diff
@@ -4963,6 +5057,12 @@
                 }
               ]
             }
+          },
+          {
+            "in": "query",
+            "name": "stage",
+            "description": "Component stage.",
+            "type": "string"
```

Comment:
>string [](http://example.com/codeflow?start=21&length=6)

Same here, should this type be an enum?

Reply:
Please see my reply here:
https://github.com/Azure/azure-rest-api-specs/pull/24051#discussion_r1201270733